spec/features/welcome_spec.rb

```diff
@@ -21,7 +21,7 @@
     click_button 'Search'
     expect(page).to have_text("car tax")
     within("table.results-table") do
-      expect(page).to have_css("tr.table-row", count: 22)
+      expect(page).to have_css("tr.table-row", count: 23)
```

Comment:
This is a terrible test. 

Ideally, we shouldn't test on CSS class as they can change. 
Also the number 23 feels like a magic number
What exactly are should this part of the test prove? can we rewrite it so it doesn;t need an explanation?

Reply:
it may be worth just removing this test, all it really does is show that you get a result. I'm not sure it's useful anymore.

Reply:
If it isn't adding value, and you feel the other tests give enough coverage then delete.. delete.. delete..